core/src/main/java/com/datastax/oss/driver/api/core/config/DefaultDriverOption.java

```diff
@@ -808,6 +807,14 @@
    * <p>Value-type: int
    */
   SESSION_LEAK_THRESHOLD("advanced.session-leak.threshold"),
+  /**
+   * The time after the node level metrics will be evicted. The eviction will happen only if none of
+   * the enabled node-level metrics is updated for a given node for a given time. When the
+   * eviction-time will elapse, all metrics for a given node are removed.
```

Comment:
```suggestion
   * the enabled node-level metrics is updated for a given node within this time window.
```

The last sentence seems redundant.